test/datasets_utils.py

```diff
@@ -572,35 +573,42 @@ def test_transforms_v2_wrapper(self, config):
 
         try:
             with self.create_dataset(config) as (dataset, info):
-                for target_keys in [None, "all"]:
-                    if target_keys is not None and self.DATASET_CLASS not in {
-                        torchvision.datasets.CocoDetection,
-                        torchvision.datasets.VOCDetection,
-                        torchvision.datasets.Kitti,
-                        torchvision.datasets.WIDERFace,
-                    }:
-                        with self.assertRaisesRegex(ValueError, "`target_keys` is currently only supported for"):
-                            wrap_dataset_for_transforms_v2(dataset, target_keys=target_keys)
-                        continue
-
-                    wrapped_dataset = wrap_dataset_for_transforms_v2(dataset, target_keys=target_keys)
-                    assert isinstance(wrapped_dataset, self.DATASET_CLASS)
-                    assert len(wrapped_dataset) == info["num_examples"]
-
-                    wrapped_sample = wrapped_dataset[0]
-                    assert tree_any(
-                        lambda item: isinstance(item, (datapoints.Datapoint, PIL.Image.Image)), wrapped_sample
-                    )
+                wrap_dataset_for_transforms_v2(dataset)
```

Comment:
This is basically the same test, but a little restructured, since we now also test the serialized version of the dataset.